app/src/main/java/com/framgia/fsalon/screen/bill/BillActivity.java

```diff
@@ -11,13 +13,19 @@
 import com.framgia.fsalon.data.source.remote.ServiceRemoteDataSource;
 import com.framgia.fsalon.data.source.remote.StylistRemoteDataSource;
 import com.framgia.fsalon.databinding.ActivityBillBinding;
+import com.framgia.fsalon.screen.login.LoginActivity;
```

Comment:
ERROR: Unused import: com.framgia.fsalon.screen.login.LoginActivity. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports